addons/binding/org.openhab.binding.zway/META-INF/MANIFEST.MF

```diff
@@ -17,7 +17,7 @@ Import-Package:
  com.google.common.collect,
  com.google.gson,
  com.google.gson.annotations,
- com.google.gson.internal,
+ com.google.gson.internal;resolution:=optional,

```

Comment:
Why this change?

Reply:
If you want to install the binding via the addon folder you get an unsolved dependency and you have to change it to get the binding running. I also noticed that at some other bindings. It can be that it’s not a problem if it is integrated into openHAB. But by using the .jar file this change is needed.

Reply:
But then you can install it have a class not found is soon as it loading any class with an import  to `com.google.gson.internal`

Reply:
Hmm I haven’t seen any issue with an class trying to import com.google.gson.internal. 
Can you give me another hint why this can be a problem.

Reply:
So I checked again, and starting from eclipse is working fine without optional. But for the single .jar file it is required or the binding wouldn't start.
So I think it could be removed for the PR.
tdyt?

Reply:
I removed the optional for the PR. From the Eclipse it works fine without it